pkg/csi_driver/controller.go

```diff
@@ -241,7 +242,9 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 	filer, err := s.config.fileService.GetInstance(ctx, newFiler)
 	// No error is returned if the instance is not found during CreateVolume.
 	if err != nil && !file.IsNotFoundErr(err) {
-		return nil, file.StatusError(err)
+		// Failed to GetInstance, however the Filestore instance may already be created.
+		// The error should be non-final.
+		return nil, common.NewTemporaryError(codes.Unavailable, fmt.Errorf("CreateVolume, failed to get instance: %w", err))
```

Comment:
we are at the CSI function level (CreateVolume) so we shouldn't directly return temporary error to the caller 

```
Do not return `TemporaryError` directly
// from CSI Spec API calls, as CSI Spec API calls MUST return a
// standard gRPC status.
```

Reply:
^ please fix.